lib/OpenQA/Worker/Commands.pm

```diff
@@ -88,15 +88,16 @@ sub websocket_commands {
     elsif ($msg eq 'livelog_start') {
         # change update_status timer if $job running
         if (backend_running) {
-            $OpenQA::Worker::Jobs::do_livelog = 1;
-            change_timer('update_status', STATUS_UPDATES_FAST);
+            print "starting livelog\n" if $verbose;
+            $OpenQA::Worker::Jobs::do_livelog += 1;
```

Comment:
I guess we need to trigger an update_status here directly - or it can take quite a while for the liveview to get data